brave/brave-core
# Added back Chromium implementation of views to look for individual cookies stored per-website.

Resolves https://github.com/brave/brave-browser/issues/27150

Added back Chromium implementation of views to look for individual cookies stored per-website.

https://user-images.githubusercontent.com/2965009/228226221-1b991bba-53e4-442c-b37a-aa3017390423.mp4



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
- Go to `brave://settings/cookies/detail?site=[domain]` and check cookies for this site can be reviewd and deleted
- Check cookies editing page can be navigated from `brave://settings/content/siteDetails` by clicking to `Cookies and Site Data` link
